cmd/gangway/handlers.go

```diff
@@ -297,7 +299,28 @@ func generateInfo(w http.ResponseWriter, r *http.Request) *userInfo {
 	defer file.Close()
 	caBytes, err := ioutil.ReadAll(file)
 	if err != nil {
-		log.Warningf("Could not read CA file: %s", err)
+		log.Errorf("Could not read CA file: %s", err)
+		http.Error(w, "Could not read CA file", http.StatusInternalServerError)
+		return nil
```

Comment:
This change make `go test` failed. Consider leave it as it was _or_ update the testing code.